Vagrant encountered an unexpected communications error with the Vagrant VMware Utility driver.

First of all, I am sorry for posting this issue -- I know how many times it has been posted in the past.

I am trying to run Vagrant on macOS 11.6 (Big Sur) on an M1 processor. Since VirtualBox won't work here, I am using the VMWare Fusion provider. I have VMWare Fusion Tech Preview, which I believe is the only version that will work. 

```
% vagrant up --provider vmware_desktop
Vagrant encountered an unexpected communications error with the
Vagrant VMware Utility driver. Please try to run the command
again. If this error persists, please contact support@hashicorp.com
```

```
% sudo launchctl list | grep com.vagrant.vagrant-vmware-utility
-       0       com.vagrant.vagrant-vmware-utility-stopper
-       1       com.vagrant.vagrant-vmware-utility
```

```
% sudo cat "/Library/Application Support/vagrant-vmware-utility/service.log"
2021-10-21T22:47:31.159-0500 [INFO]  vagrant-vmware-utility.api: created: vmx=<nil>
2021-10-21T22:47:31.159-0500 [ERROR] vagrant-vmware-utility.api: path loading failure: error="Failed to locate VMware installation directory!"
2021-10-21T22:47:31.159-0500 [ERROR] vagrant-vmware-utility.api: base driver setup failure: error="Failed to locate VMware installation directory!"
Failed to setup Vagrant VMWare API service - Failed to locate VMware installation directory!
```

This log seems different from what I've seen here. Am I doing something wrong?

Thanks in advance.

--- a/website/content/docs/providers/vmware/faq.mdx
+++ b/website/content/docs/providers/vmware/faq.mdx
@@ -8,6 +8,19 @@ description: |-
 
 # Frequently Asked Questions
 
+## Q: How do I use the VMware Fusion Tech Preview?
+
+The Vagrant VMware utility expects VMware Fusion to be installed in a specific
+path on macOS. Since the VMware Fusion Tech Preview installs into a different
+path, the Vagrant VMware utility will fail to detect the installation and will
+not start. To resolve this you can create a symlink from the VMware Fusion
+Tech Preview installation path to the normal VMware Fusion installation path:
+
+```shell-session
+sudo ln -s /Applications/VMware\ Fusion\ Tech\ Preview.app /Applications/VMware\ Fusion.app
+```
+
+
 ## Q: How do I upgrade my currently installed Vagrant VMware plugin?
 
 You can update the Vagrant VMware plugin to the latest version by re-running the

